tools/operator/launcher/networkspec.yaml

```diff
@@ -2,29 +2,29 @@
 #!
 #! SPDX-License-Identifier: Apache-2.0
 ---
-#! fabric_version:
+#! fabricVersion:
 #! Released images are pulled from docker hub hyperledger/, e.g. 1.4.1 or 2.0.0
 #! Development stream images are pulled from
 #! nexus3.hyperledger.org:10001/hyperledger/, e.g. 1.4.1-stable or 2.0.0-stable
-fabric_version: 1.4.4-stable
+fabricVersion: 1.4.4-stable
```

Comment:
This should be 1.4.5-stable.
Maybe you need to rebase.

Reply:
I will change it to 1.4.5

Reply:
will change to 1.4.5